todo.sh

```diff
@@ -113,6 +113,8 @@ $indentedJoinedConfigFileLocations
 		        Don't auto-archive tasks automatically on completion
 		    -A
 		        Auto-archive tasks automatically on completion
+		    -m
+		        Maintain priority field on completion
```

Comment:
I think the "maintain" is meant to establish a mnemonic for the chosen option letter, but it sounds a bit confusing to my non-native ears. I'd prefer _Don't remove a priority on completion_.